chart/values.yaml

```diff
@@ -325,6 +325,7 @@ etcd:
 
   auth:
     rbac:
+      create: false
```

Comment:
What is this for?

Reply:
This was needed to disable auth otherwise etcd-client doesn't work in core-agent.

Reply:
Ok